raiden_contracts/deploy/__main__.py

```diff
@@ -537,7 +539,7 @@ def verify(ctx, rpc_provider, contracts_version, flavor):
     web3.middleware_stack.inject(geth_poa_middleware, layer=0)
     print('Web3 provider is', web3.providers[0])
 
-    contract_manager = ContractManager(contracts_precompiled_path(contracts_version))
+    contract_manager = ContractManager(flavor_enum, contracts_precompiled_path(contracts_version))
```

Comment:
I see what you are trying to do here with `flavor_of_lower_name[flavor]` but there is a better way. To turn a CLI argument into an enum look at what we do in Raiden for `RaidenNetworkType` [here](https://github.com/raiden-network/raiden/blob/4cf30f351455ca09c4800a402a5da0ed7042fa71/raiden/ui/cli.py#L158).

We are using a custom click choice defined [here](https://github.com/raiden-network/raiden/blob/4cf30f351455ca09c4800a402a5da0ed7042fa71/raiden/utils/cli.py#L261)

Reply:
I see.  I sometimes need to read `unlimited` from JSON files and turn that into `Flavor.UNLIMITED`.  So anyway I need some conversions.